README.md

```diff
@@ -963,6 +964,41 @@ foreach ($devices->iterateAllElements() as $device) {
 $ composer require google/cloud-iot
 ```
 
+## Google Cloud Redis (Alpha)
+
+- [API Documentation](http://googlecloudplatform.github.io/google-cloud-php/#/docs/latest/redis/readme)
+- [Official Documentation](https://cloud.google.com/redis/docs)
+
+#### Preview
+
+```php
+require 'vendor/autoload.php';
+
+use Google\Cloud\Redis\V1beta1\CloudRedisClient;
+
+$client = new CloudRedisClient();
+
+$projectId = '[MY_PROJECT_ID]';
+$location = '-';
```

Comment:
Just a little nit: Could this be a template, like above?

Reply:
In other places we use `us-central1` for location, should we do this here? (`-` is also a valid value for Redis here)

Reply:
Let's leave it as-is - the `-` token means any region, so is perhaps more useful to more people. I will add a comment though!